firehydrant/rotations.go

```diff
@@ -2,6 +2,7 @@ package firehydrant
 
```

Comment:
Since we moved to new client, can this go?

Reply:
removed!